docs/reference/hooks.md

```diff
@@ -113,24 +113,26 @@ Hooks for building new areas of the admin interface (alongside pages, images, do
 Add or remove panels from the Wagtail admin homepage. The callable passed into this hook should take a `request` object and a list of panel objects and should modify this list in place as required. Panel objects are [](template_components) with an additional `order` property, an integer that determines the panel's position in the final ordered list. The default panels use integers between `100` and `300`.
 
 ```python
-from django.utils.safestring import mark_safe
-
+from django.utils.html import format_html
 from wagtail.admin.ui.components import Component
 from wagtail import hooks
 
 class WelcomePanel(Component):
     order = 50
 
     def render_html(self, parent_context):
-        return mark_safe("""
-        <section class="panel summary nice-padding">
-          <h3>No, but seriously -- welcome to the admin homepage.</h3>
-        </section>
-        """)
+        return format_html(
+            """
+            <section class="panel summary nice-padding">
+              <h3>No, but seriously -- welcome to the admin homepage.</h3>
+            </section>
+            """
+        )
```

Comment:
Best to avoid changes to other areas of the docs, this may not be the nicest formatting but if you think it's something to fix, please can you prepare a separate PR with just this change.

Reply:
Actually, sorry I realise you changed it to format_html, ignore my comment. let's add this.